src/main/resources/doc/en/html/guide/prefs/pref-intl.html

```diff
@@ -61,7 +61,7 @@ <h1>
             </table>
           </center>
           <p>
-            Because the <i>IEC</i> style tends to be more popular in the U.S., while the <i>ANSI<(i>) style tends to be more popular in Europe, some people refer to these styles according to these regions. <!-- The DIN 40700 standard was a standard for drafting digital and analog electronic components adopted by DIN, a German standards organization. DIN adopted the rectangular standard for digital components in 1976, but some engineers continue to use the older style; they appear to be increasingly rare.-->
+            Because the <i>IEC</i> style tends to be more popular in the U.S., while the <i>ANSI</i> style tends to be more popular in Europe, some people refer to these styles according to these regions. <!-- The DIN 40700 standard was a standard for drafting digital and analog electronic components adopted by DIN, a German standards organization. DIN adopted the rectangular standard for digital components in 1976, but some engineers continue to use the older style; they appear to be increasingly rare.-->
```

Comment:
IEC and ANSI are mixed up here! It should be:

"Because the <i>ANSI</i> style tends to be more popular in the U.S., while the <i>IEC</i> style tends to be more popular in Europe, some people refer to these styles according to these regions."

Please check whether other parts/localisations are affected, too!

Reply:
As before